tests/LibrdkafkaTestApp.py

```diff
@@ -66,10 +67,34 @@ def __init__(self, cluster, version, conf=None,
 
             elif mech == 'OAUTHBEARER':
                 self.security_protocol = 'SASL_PLAINTEXT'
-                conf_blob.append('enable.sasl.oauthbearer.unsecure.jwt=true\n')
-                conf_blob.append(
-                    'sasl.oauthbearer.config=%s\n' %
-                    self.conf.get('sasl_oauthbearer_config'))
+                oidc = cluster.find_app(OauthbearerOIDCApp)
+                if oidc:
+                    conf_blob.append('sasl.oauthbearer.method=%s\n' %
+                                     self.conf.get('sasl.oauthbearer.method'))
```

Comment:
`self.conf` is trivup configuration, which is for configuring trivup itself. It uses underscores as config parameter keys.
`conf_blob` is the client config being constructed.

These should be created and added to the client configuration by trivup, which they are. So just rely on trivup providing these.